roles/ceph-mon/defaults/main.yml

```diff
@@ -4,18 +4,12 @@
 ###########
 # GENERAL #
 ###########
-
 mon_group_name: mons
 
 # ACTIVATE BOTH FSID AND MONITOR_SECRET VARIABLES FOR NON-VAGRANT DEPLOYMENT
 monitor_secret: "{{ monitor_keyring.stdout }}"
 admin_secret: 'admin_secret'
 
-# CephFS
-cephfs_data: cephfs_data
-cephfs_metadata: cephfs_metadata
-cephfs: cephfs
```

Comment:
I see here you removed cephfs variable although it's still used here https://github.com/ceph/ceph-ansible/pull/2036/files#diff-1ebfa6bd1a372462e995257b0e1f09e3R9

Reply:
good catch, this should remain